recipe/meta.yaml

```
@@ -1,33 +1,46 @@
 {% set name = "semantic_version" %}
-{% set version = "2.8.5" %}
+{% set version = "2.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d2cb2de0558762934679b9a104e82eca7af448c9f4974d1f3eeccff651df8a54
+  sha256: bdabb6d336998cbb378d4b9db3a4b56a1e3235701dc05ea2690d9a997ed5041c
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
-  noarch: python
+  skip: True  # [py<35]
```

https://inspector.pypi.io/project/semantic-version/2.10.0/packages/7d/31/f2289ce78b9b473d582568c234e104d2a342fd658cc288a7553d83bb8595/semantic_version-2.10.0.tar.gz/semantic_version-2.10.0/setup.cfg#line.34
```suggestion
  skip: True  # [py<27]
```